controllers/gitrepository/gitrepository_webhook_controller.go

```diff
@@ -42,11 +41,10 @@ import (
 type Reconciler struct {
 	client.Client
 	log      logr.Logger
-	Scheme   *runtime.Scheme
 	recorder record.EventRecorder
 }
 
-//+kubebuilder:rbac:groups=devops.kubesphere.io,resources=webhooks,verbs=get;list
+//+kubebuilder:rbac:groups=devops.kubesphere.io,resources=webhooks,verbs=get;list;update;patch
```

Comment:
Do we need to execute `controller-gen` to update RBAC manifest after we change the RBAC rule?

Reply:
Sure.